### ISSUE TITLE
[Filesystem] missing info on current directory

### ISSUE THREAD

**Issue by xosofox** [symfony/symfony-docs#9674]
Going through the docs, I'm missing the point where there is any information about the "current working directory" when looking at the relative examples´, like `$fileSystem->exists(array('rabbit.jpg', 'bottle.png'));`

It's not like the examples are self-explanatory and "ready to be used" without this important bit.

This is not a "Guys, fix this!" issue - I'm looking at sending a PR with a proposed clarification; I just like to have this recorded

**Comment by javiereguiluz** [comment]
I agree. The current docs are lacking:

```rst
// this directory exists, return true
$fileSystem->exists('/tmp/photos');

// rabbit.jpg exists, bottle.png does not exist, return false
$fileSystem->exists(array('rabbit.jpg', 'bottle.png'));
```

What if we tweak the comments to explain things better? For example:

```rst
// if this absolute directory exists, returns true
$fileSystem->exists('/tmp/photos');

// non-absolute paths are relative to the directory where the running PHP
// script is stored. If rabbit.jpg exists and bottle.png does not exist, returns false
$fileSystem->exists(array('rabbit.jpg', 'bottle.png'));
```

**Comment by xosofox** [comment]
I'd even go as far as to make this a separate note as this is valid for all actions and not something to be a side note in another example. So Maybe add a Note:

.. note::
    If you are using non-absolute file names, the paths are relative to the directory where the running PHP 
   script is stored

**Comment by javiereguiluz** [comment]
Fixed by #9708.
